Formula/ignition-gui4.rb

```diff
@@ -16,10 +16,10 @@ class IgnitionGui4 < Formula
   depends_on "pkg-config" => [:build, :test]
   depends_on "ignition-cmake2"
   depends_on "ignition-common3"
-  depends_on "ignition-msgs5"
+  depends_on "ignition-msgs6"
```

Comment:
Broke the changes to this file into https://github.com/osrf/homebrew-simulation/pull/1068, with an updated source code commit that expects these dependencies.